views/007_events.sql

```diff
@@ -37,6 +37,7 @@ CREATE OR REPLACE FUNCTION insert_responder_in_event_queue() RETURNS TRIGGER AS
 BEGIN
     IF TG_OP = 'INSERT' THEN
         INSERT INTO event_queue(name, properties) VALUES ('incident.responder.added', jsonb_build_object('id', NEW.id));
+        INSERT INTO event_queue(name, properties) VALUES ('incident.responder.added.sync', jsonb_build_object('id', NEW.id));
```

Comment:
Why do we need this? Why would any of the incident.responder.added events be handled async?

Reply:
It's notifying 3rd party responder clients: Jira & MSPlanner.